src/query/generated/proto/admin/namespace.proto

```diff
@@ -49,3 +49,20 @@ message NamespaceSchemaResetRequest {
 
 message NamespaceSchemaResetResponse {
 }
+
+// NamespaceReadyRequest is the request for transitioning a namespace
+// to the ready state.
+message NamespaceReadyRequest {
```

Comment:
might be nice to document this endpoint/request as idempotent so clients can safely retry any errors